starcounter-include.html

```diff
@@ -315,6 +315,16 @@
             return range.createContextualFragment(htmlStr);
         }
 
+        function findCompositionProvider(obj) {
```

Comment:
How did you measure it?

Should we also observe the changes in runtime? as theoretically the position of this property may change in fly.

I think it's worth adding benchmark as we will perform this operation 2x number of partials on page load, then on every change of property of every partial view-model.